models/endoscopic_tool_segmentation/large_files.yml

```diff
@@ -1,9 +1,9 @@
 large_files:
   - path: "models/model.pt"
-    url: "https://drive.google.com/file/d/19yS3t2oLBiB7wT-qeQ82da95VJs_vzRK/view?usp=share_link"
-    hash_val: "9d568aaf7188b7b3f254f87c8656d7fe"
+    url: "https://drive.google.com/file/d/1_VbLCkQBbcHC1I38aDN4nkomVOJYpGCW/view?usp=sharing"
```

Comment:
Hi @binliunls , thanks for the changes. Are the weights here reproducible? If not, may still need to retrain it after #363 is merged.

Reply:
Hi @yiheng-wang-nv ,
    Thank you for the info. This is just the first drop with private dataset. I will keep updating this bundle until it achieve the previous IoU and will update it after the #363 merged.

Thanks,
Bin